docs/configurations.md

```diff
@@ -1,53 +1,70 @@
 # ScalarDB Configurations
 
-This document describes the configurations for ScalarDB.
+This page describes the available configurations for ScalarDB.
 
-## Transaction manager configurations
+## Transaction managers
 
-ScalarDB has several transaction manager implementations, such as Consensus Commit, gRPC, and JDBC, and you can specify one of the implementations with the `scalar.db.transaction_manager` property.
-This section describes the configurations for each transaction manager.
+Implemented within ScalarDB are three transaction managers: Consensus Commit, gRPC, and JDBC. You can specify one of the transaction managers by using the `scalar.db.transaction_manager` property and then specify the configurations for the transaction manager.
 
 ### Consensus Commit
 
-Consensus Commit is the default transaction manager in ScalarDB.
-If you do not specify the `scalar.db.transaction_manager` property, or if you specify `consensus-commit` for the property, Consensus Commit will be used.
+Consensus Commit is the default transaction manager type in ScalarDB. To use the Consensus Commit transaction manager, add the following to the ScalarDB properties file:
 
-This section describes the configurations for Consensus Commit.
+```properties
+scalar.db.transaction_manager=consensus-commit
+```
+
+{% capture notice--info %}
+**Note**
+
+If you don't specify the `scalar.db.transaction_manager` property, `consensus-commit` will be the default value.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
 #### Basic configurations
 
-The basic configurations for Consensus Commit are as follows:
+The following basic configurations are available for Consensus Commit transactions:
```

Comment:
```suggestion
The following basic configurations are available for Consensus Commit:
```
@josh-wong I think it should probably be `Consensus Commit` or `the Consensus Commit transaction manager` since it's the configuration of the component.
By contrast, `Consensus Commit transactions` sound like transactions that the transaction manager executes.
What do you think?

Reply:
Thank you for the explanation! That makes sense to me that Consensus Commit is strictly a transaction manager rather than a type of executed transaction.